src/content/language/constructors.md

```diff
@@ -111,34 +91,28 @@ class Point {
 }
 ```
 
-Remember that constructors are not inherited, which means that a
-superclass's named constructor is not inherited by a subclass. If you
-want a subclass to be created with a named constructor defined in the
-superclass, you must implement that constructor in the subclass.
+A subclass doesn't inherit a superclass's named constructor.
+To create a subclass with a named constructor defined in the superclass,
+implement that constructor in the subclass.
 
+### Non-default superclass constructors
 
-## Invoking a non-default superclass constructor
+Dart executes constructors in the following order:
 
-By default, a constructor in a subclass calls the superclass's unnamed,
-no-argument constructor.
-The superclass's constructor is called at the beginning of the
-constructor body. If an [initializer list](#initializer-list)
-is also being used, it executes before the superclass is called.
-In summary, the order of execution is as follows:
-
-1. initializer list
-1. superclass's no-arg constructor
+1. [initializer list](#use-an-initializer-list)
+1. superclass's unnamed, no-arg constructor
 1. main class's no-arg constructor
```

Comment:
This bullet point feels out of place - that's not a new problem in this PR though.

I don't know why we need to talk specifically about the arguments to the main class constructor - this description applies to any constructor definition that doesn't have a super constructor invocation.

I was trying to find a suggestion for a rewrite of this section and it struck me that waiting until the end of the doc to show what an "initializer list" is may be hurting us.

I wonder if we should try to show syntax and terminology near the top of this doc. If a reader knows that the "initializer list" is "the part with `fieldName = parameterName` because they saw that syntax labeled, I think they would have an easier time understanding what it means for the "initializer list" to run in a given position in the instantiation process, even if they don't know the details that will be introduced later.

Reply:
Understood. I changed the first example to call out what the initializer list was.